R/plot.R

```diff
@@ -197,69 +320,69 @@ RadialSUCRA <- function(SUCRAData, ColourData, BUGSnetData, colourblind=FALSE) {
     dat.edges$n.stud[comp.i+1] <- edges$edge.weight[i]
     dat.edges$SUCRA[comp.i] <- SUCRA$SUCRA[SUCRA$Treatment == edges$from[i]]
     dat.edges$SUCRA[comp.i+1] <- SUCRA$SUCRA[SUCRA$Treatment == edges$to[i]]
-    dat.edges$lwdO[comp.i] <- lwd.minO + (edges$edge.weight[i] - study_min)*(lwd_rangeO/study_range)
-    dat.edges$lwdA[comp.i] <- lwd.minA + (edges$edge.weight[i] - study_min)*(lwd_rangeA/study_range)
-    dat.edges$lwdO[comp.i+1] <- lwd.minO + (edges$edge.weight[i] - study_min)*(lwd_rangeO/study_range)
-    dat.edges$lwdA[comp.i+1] <- lwd.minA + (edges$edge.weight[i] - study_min)*(lwd_rangeA/study_range)
+    dat.edges$lwdO[comp.i] <- lwd.minO + (edges$edge.weight[i] - study_min) * (lwd_rangeO / study_range)
+    dat.edges$lwdA[comp.i] <- lwd.minA + (edges$edge.weight[i] - study_min) * (lwd_rangeA / study_range)
+    dat.edges$lwdO[comp.i+1] <- lwd.minO + (edges$edge.weight[i] - study_min) * (lwd_rangeO / study_range)
+    dat.edges$lwdA[comp.i+1] <- lwd.minA + (edges$edge.weight[i] - study_min) * (lwd_rangeA / study_range)
     comp.i <- comp.i + 2
     ID <- ID + 1
   }
   # add lines #
   CreateNetwork <- function(Type) {
```

Comment:
Should this inner function be factored out and documented?

Reply:
I'm loathe to make a "big" change like this. This task was only really to add headers, and I've already done a lot of formatting in addition. I saw plenty of things that could be improved as I was going through the code, so I'd like to leave those for tasks for that purpose.

Reply:
Ok, you can leave them where they are, but can you add roxygen comments for these functions though please?